packages/uswds-core/src/styles/_notifications.scss

```diff
@@ -105,6 +105,16 @@ $uswds-notifications:
 + "\A   If your project configures disabled color settings"
 + "\A   or uses disabled color tokens, you probably need to update your"
 + "\A   code. Details: github.com/uswds/uswds/releases/tag/v3.6.0"
++ "\A --------------------------------------------------------------------"
++ "\A 3.7.0"
++ "\A Details: github.com/uswds/uswds/releases/tag/v3.7.0"
++ "\A "
++ "\A - Accessibility: [usa-identifier] We updated the markup to improve"
++ "\A   how screen readers read the phrase 'An official'."
++ "\A - Accessibility: [usa-range] We updated the markup to remove"
++ "\A   redundant ARIA attributes and improve the screen reader experience."
++ "\A - Accessibility: [usa-range] We added optional data attributes that"
++ "\A   allow adding units to screen reader output."
```

Comment:
Did we want to include a note about updating the banner flag icon?

Reply:
I'm not sure that one rises to the level of a notification since what they get by default (an optimized PNG) is arguably the better solution for the time being